compose/src/main/java/com/maplibre/compose/ramani/MapLibreCompose.kt

```diff
@@ -172,6 +177,22 @@ internal class MapApplier(
             zIndexReferenceAnnotationManagerMap[zIndex] = circleManager
         }
 
+        circleManager.addClickListener(OnCircleClickListener { annotation ->
+            decorations.findInputCallback<CircleNode, Circle, Unit>(
+                nodeMatchPredicate = { it.circle.id == annotation?.id && it.circleManager.layerId == circleManager.layerId },
+                nodeInputCallback = { onTap }
```

Comment:
Can we just do `nodeInputCallback = onTap`? (Same idea on L191, 265, and 273).

Reply:
Unfortunately not, the brackets correctly capture the value which is associated with the `CircleNode`, `SymbolNode`, etc.

Reply:
Oh, fun....